ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -638,18 +638,18 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce
-- as: aws-ipi-ovn-hypershift-mgmt-critical-f7
-  cron: 19 5 8,15,22,29 * *
+- as: aws-ipi-ovn-hypershift-mgmt-critical-f7-longduration
+  cron: 33 4 2,9,16,25 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       TEST_ADDITIONAL: Hypershift|Network_Observability
       TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;HyperShiftMGMT&
       TEST_IMPORTANCE: Critical
-      TEST_TIMEOUT: "35"
+      TEST_TIMEOUT: "90"
     test:
-    - chain: openshift-e2e-test-hypershift-qe-mgmt
+    - chain: openshift-e2e-test-hypershift-qe-mgmt-longduration
```

Comment:
I think `longduration` are mean to run test cases with `longduration`.
If your test cases do not have that tag, and you just want longer execution time, overwrite `TEST_TIMEOUT`, but not change the test name and test chain.

Reply:
@liangxia Okay, updated.

Reply:
Perhaps update the PR title, too ?

Reply:
Done.